skynet/env.py

```diff
@@ -101,7 +101,7 @@ def tobool(val: str | None):
 
 # summaries
 summary_minimum_payload_length = int(os.environ.get('SUMMARY_MINIMUM_PAYLOAD_LENGTH', 100))
-enable_batching = tobool(os.environ.get('ENABLE_BATCHING', 'true'))
+enable_batching = tobool(os.environ.get('ENABLE_BATCHING', 'true' if use_vllm else 'false'))
```

Comment:
maybe `str(use_vllm)`?

Reply:
i assume it does not have anything to do with oci but rather with local inference with ollama?

Reply:
Yeah it's for ollama. Good point on the simplification.